packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -539,6 +508,10 @@ export interface Android {
      * Local path or remote URL to an image to use for your app's icon on Android. If specified, this overrides the top-level `icon` and the `android.icon` keys. Should follow the [specified guidelines](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive). This icon will appear on the home screen.
      */
     foregroundImage?: string;
+    /**
```

Comment:
Any reason why this comment was changed? I wrote my orignal comment to be more consistent with the previous ones.

@EvanBacon